src/sns-adapter.ts

```diff
@@ -101,10 +104,15 @@ export class SNSAdapter implements ISNSAdapter {
             if (req.is("text/plain")) {
                 event = createSnsEvent(event.TopicArn, "EXAMPLE", event.Subject || "", event.Message, createMessageId(), event.MessageAttributes || {});
             }
-            getHandler()(event, this.createLambdaContext(fn), (data) => {
+            const sendIt = (data) => {
                 res.send(data);
                 process.env = oldEnv;
-            });
+                this.sent(data);
```

Comment:
this is really just for testing; i'd appreciate suggestions on how to do this better